src/CoreBundle/Field/FieldSidebarConnected.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Field;
+
+use ChameleonSystem\CoreBundle\DataAccess\MenuItemDataAccessInterface;
+use ChameleonSystem\CoreBundle\DataModel\MenuCategoryAndItem;
+use ChameleonSystem\CoreBundle\ServiceLocator;
+
+class FieldSidebarConnected extends \TCMSFieldVarchar
+{
+    /**
+     * @var MenuItemDataAccessInterface
+     */
+    private $menuItemDataAccess;
+
+    public function __construct(?MenuItemDataAccessInterface $menuItemDataAccess = null)
+    {
+        // TODO isVirtual?
+
+        if (null === $menuItemDataAccess) {
+            $menuItemDataAccess = ServiceLocator::get('chameleon_system_core.sidebar.menu_access');
+        }
+        $this->menuItemDataAccess = $menuItemDataAccess;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function GetReadOnly(){
+        $menuPath = $this->getMenuPath($this->recordId);
+
+        return sprintf('<div class="form-control form-control-sm" readonly>%s</div>', $menuPath);
+    }
+
+    private function getMenuPath(string $tableId): string
+    {
+        $menuEntry = $this->getMatchingEntryFromMenu($tableId);
```

Comment:
There could be multiple assignments. Why only show one?

Reply:
Showing multiple would make it more complicated.

Also there are hardly cases with mutliple menu entries for just one table.

One use-case for this function here would be to be able to easily access the datasets of a table in the backend. For this one path (entry here) is enough.